crates/weaver_resolver/data/registry-test-4-events/expected-registry.json

```diff
@@ -1,15 +1,160 @@
 {
   "registry_url": "https://127.0.0.1",
   "groups": [
+    {
+      "id": "browser.test.event_with_fields",
+      "type": "event",
+      "brief": "An event that adds global attributes for reuse.\n",
+      "prefix": "browser",
+      "attributes": [
+        0,
+        1,
+        2,
+        3
+      ],
+      "name": "browser.test.event_with_fields",
+      "lineage": {
+        "source_file": "data/registry-test-4-events/registry/browser-event.yaml"
+      },
+      "body": {
+        "name": "browser.test.event_with_body.fields",
```

Comment:
Linking for the comment in the AnyValue PR (https://github.com/open-telemetry/weaver/pull/365#discussion_r1767768808) these "name" entries become `id`

```json
      "body": {
        "id": "browser.test.event_with_body_details.fields",
        "type": "map",
        "brief": "A map of fields that are not referenced in the attributes",
        "note": "This map is not referenced in the attributes",
        "stability": "experimental",
        "examples": [
          "{ \"some.field\": \"some value\" }"
        ],
        "requirement_level": "required",
        "fields": [
          {
            "id": "some.field",
            "type": "string",
```